boxoffice/templates/ticket_assigment.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='UTF-8'>  
+  <title>{{ org.title }}</title>  
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:400italic,600italic,400,600" />
+  <style type="text/css">
+    * { 
+      margin: 0; 
+      padding: 0; 
+    }
+
+    body {
+      font-family: "Source Sans Pro",Arial,sans-serif;
+      font-size: 16px;
+      line-height: 1.2;
+      color: #333;
+      background-color: #fff
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    p { 
+      margin: 5px 0 10px;
+    }
+
+    .button {
+      border-width: 0px 0px 3px;
+      border-style: solid;
+      border-radius: 5px;
+      padding: 6px 12px;
+      color: #fff;
+      font-size: 15px;
+      display: inline-block;
+      background-color: #5BC0DE;
+      border-color: #46B8DA;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+
+    <p>Hello {{ line_item.current_assignee.fullname }},</p>
+
+    <p>Thank you for filling in the details. {{line_item.item.title}} has been assigned to you.</p>
```

Comment:
Do we need 'Thank you for filling in the details.'. One possibility is, the assignee may not have filled the details themselves.